### PR TITLE
feat: Add search page

### PR DESCRIPTION
**Pull request by NataliiaKononikhina**

### CODE REVIEW

**Comment by paszkowskiDamian** [comment]
Does Search page without any results supposed to look like this?
image

It covers an entire screen.

**Comment by paszkowskiDamian** [comment]
image

Please change page title.

File: user/themes/centra/scss/_navigation.scss [hunk 1]
```diff
@@ -61,11 +61,15 @@ $menu-breakpoint: 715px;
   display: none;
   width: 36px;
   height: 36px;
-  background-color: #1B1B1B;
+  background-color: $blackish;
```
Comment (paszkowskiDamian) [inline user/themes/centra/scss/_navigation.scss]: nice 🤣

File: user/themes/centra/templates/partials/head.html.twig [hunk 2]
```diff
@@ -48,6 +51,10 @@
             var footer = document.getElementById("landing-page-footer");
             var bounding = footer.getBoundingClientRect();
 
+            if (!bar) {
```
Comment (paszkowskiDamian) [inline user/themes/centra/templates/partials/head.html.twig]: What?

Comment (paszkowskiDamian) [reply user/themes/centra/templates/partials/head.html.twig]: Sorry, GitHub hid `bar` declaration.

**Comment by NataliiaKononikhina** [comment]
> Does Search page without any results supposed to look like this?

https://www.figma.com/file/JkX2TJlz5J1XfRWD1Yp9R6g1/Centra-Dev-Site---v03-(IBM-body)?node-id=0%3A1

You can find design here.

File: user/themes/centra/scss/_navigation.scss (hunk 1 above)
Comment (NataliiaKononikhina) [reply user/themes/centra/scss/_navigation.scss]: We already had this variable :sweat_smile:

**Pull request merged**